docs/conf.py

```diff
@@ -1,42 +1,46 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(0, os.path.abspath(".."))  # pylint: disable=wrong-import-position
```

Comment:
Why are you not solving the problem?

Reply:
I think this is required in order to import the version on the next line, I remember looking into this some time ago and this was the cleanest solution I came up with